packages/components/src/Button/Button.css

```diff
@@ -146,9 +146,20 @@
   padding: 0;
 }
 
+.onlyIcon.small {
+  width: calc(var(--space-small) * 3);
+}
+
+.onlyIcon.base {
+  width: calc(var(--space-base) * 2.25);
+}
+
+.onlyIcon.large {
+  width: calc(var(--space-large) * 2.25);
```

Comment:
If the multiplier here is set to `2` instead of `2.25` then we get the 48x48 (1:1) ratio that we're looking for - otherwise this looks great!

Reply:
oh yeah.. done